ordered_set.py

```diff
@@ -249,35 +270,35 @@ def discard(self, key):
                 if v >= i:
                     self.map[k] = v - 1
 
-    def clear(self):
+    def clear(self) -> None:
         """
         Remove all items from this OrderedSet.
         """
         del self.items[:]
         self.map.clear()
 
-    def __iter__(self):
+    def __iter__(self) -> Iterator[T]:
         """
         Example:
             >>> list(iter(OrderedSet([1, 2, 3])))
             [1, 2, 3]
         """
         return iter(self.items)
 
-    def __reversed__(self):
+    def __reversed__(self) -> Iterator[T]:
```

Comment:
I think this should be `Iterable[T]`.

Reply:
The Python documentation states that:
> It should return a new iterator object that iterates over all the objects in the container in reverse order.

So, I guess it's correct.